src/game/client/components/skins.cpp

```diff
@@ -415,20 +419,23 @@ int CSkins::FindImpl(const char *pName)
 	if(str_find(pName, "/") != 0)
 		return -1;
 
-	auto d = ::find_binary(m_aDownloadSkins.all(), pName);
-	if(!d.empty())
+	CDownloadSkin DownloadNeedle;
+	mem_zero(&DownloadNeedle, sizeof(DownloadNeedle));
+	str_copy(DownloadNeedle.m_aName, pName, sizeof(DownloadNeedle.m_aName));
+	auto DownloadRange = std::equal_range(m_DownloadSkins.begin(), m_DownloadSkins.end(), DownloadNeedle);
```

Comment:
```suggestion
	auto &[RangeBegin, RangeEnd] = std::equal_range(m_DownloadSkins.begin(), m_DownloadSkins.end(), DownloadNeedle);
```

Could be more readable than `.first`, `.second`

Reply:
Done. Had to add `const` to make it compile.